src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const repairedRobot = {};
```

Comment:
inverseRobot() => inversedRobot

```suggestion
  const inversedRobot = {};
```